SUPPORTED_PLATFORMS.md

```diff
@@ -4,7 +4,7 @@
 |---|---|---|---|
 | GNU/Linux | Tier 1 | Linux >= 2.6.32 with glibc >= 2.12 | |
 | macOS | Tier 1 | macOS >= 10.7 | |
-| Windows | Tier 1 | >= Windows 7 | MSVC 2008 and later are supported |
+| Windows | Tier 1 | >= Windows 8 | VS 2015 and later are supported |
```

Comment:
Maybe VS2017 ? https://support.microsoft.com/en-us/lifecycle/search?alpha=Visual%20Studio%202015%20(all%20editions) VS 2015 seems EOL too.

Reply:
The latest update of VS2015 is still supported until 2025. For Node.js, removing support here shouldn't be an issue, as VS2015 is only supported to build addons, not Node itself.

https://support.microsoft.com/en-us/lifecycle/search?alpha=Visual%20Studio%202015%20Update%203

Reply:
Python 3's ABI is defined by "whatever VS 2015 emits", similar to Python 2 and MSVC 2008, that's why I opted for 2015 here. It's not a point of contention for me, I'm open to changing it.